app/user.go

```diff
@@ -146,7 +146,7 @@ func (c *UserClient) inviteUsers(
 	var roleIDs []string
 
 	// first get the required account role
-	role, err := getAccountRole(c.ctx, c.client, accountRole)
+	role, err := getAccountRole(c.ctx, c.client, accountRole, false)
```

Comment:
would this throw a nil pointer exception below? 

`roleIDs = append(roleIDs, role.GetId())`

Reply:
This doesn't allow nil to be returned because none is invalid for inviting a user. It'd get an error.